code/src/cljs/sixsq/nuvla/ui/utils/semantic_ui_extensions.cljs

```diff
@@ -398,7 +398,8 @@
                      :trigger    trigger
                      :on-close   (fn [& args]
                                    (when on-close
-                                     (apply on-close args))
+                                     (apply on-close args)))
+                     :on-unmount (fn []
```

Comment:
I made this change because after clicking on the `Recompute fleet` button once, if you triggered the action again quickly it looked like the previous was still in progress (spinning icon). It was because the `clicked?` flag was not reset on success, but only on explicit close of the dialog window.
I moved that part to the `onUnmount` event,which is called every time the dialog is removed from the DOM, so both on dialog close and when the action succeeds.
I tested it in a couple of other places and it does not seem to break other stuff